Havit.Blazor.Components.Web.Bootstrap/Offcanvas/OffcanvasResponsiveBreakpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Havit.Blazor.Components.Web.Bootstrap;
+
+public enum OffcanvasResponsiveBreakpoint
+{
+	/// <summary>
+	/// Offcanvas disabled, always render contents inside the viewport.
```

Comment:
I think it is the exact opposite. `None` is "always render in offcanvas", never inside viewport?

Reply:
Indeed 😅